### PR TITLE
[DONE, ready for merge; additional proof-reading/styling is welcome] Translate v2 docs in Russian. 

### PR DESCRIPTION
**Pull request by gbezyuk**
**This Pull Request reflects an ongoing work in progress.**

# Translation Progress Status: 100%

- [x] Summary (SUMMARY.md)
- [x] Installation (installation.md)
- [x] What is Vuex? (intro.md)
- [x] Getting Started (getting-started.md)

- [x] State(state.md)
- [x] Getters(getters.md)
- [x] Mutations(mutations.md)
- [x] Actions(actions.md)
- [x] Modules(modules.md)

- [x] Application Structure(structure.md)
- [x] Plugins(plugins.md)
- [x] Strict Mode(strict.md)
- [x] Form Handling(forms.md)
- [x] Testing(testing.md)
- [x] Hot Reloading(hot-reload.md)
- [x] API Reference(api.md)

# Proof-reading Progress Status: 100%

- [x] Summary (SUMMARY.md)
- [x] Installation (installation.md)
- [x] What is Vuex? (intro.md)
- [x] Getting Started (getting-started.md)

- [x] State(state.md)
- [x] Getters(getters.md)
- [x] Mutations(mutations.md)
- [x] Actions(actions.md)
- [x] Modules(modules.md)

- [x] Application Structure(structure.md)
- [x] Plugins(plugins.md)
- [x] Strict Mode(strict.md)
- [x] Form Handling(forms.md)
- [x] Testing(testing.md)
- [x] Hot Reloading(hot-reload.md)
- [x] API Reference(api.md)

### CODE REVIEW

File: docs/ru/intro.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+- Действия из разных представлений могут оказывать влияние на одни и те же части состояния приложения
+
+В рамках решения первой проблемы, передача входных параметров в глубоко вложенные компоненты может стать утомительной, а для соседних компонентов — и вовсе не помочь. Решая вторую проблему, мы нередко обнаруживаем себя прибегающими к таким средствам как обращение к прямым родителям/потомкам компонента, или в попытках внести изменения и затем синхронизировать эти изменения с другими местами в приложении, используя события. Оба подхода хрупки и быстро приводят к появлению кода, который невозможно поддерживать.
+
+Так почему бы не вынести всё общее состояние приложения из компонентов в глобальный синглтон? При использовании этого подхода, дерево компонентов превращается в одно большое "представление", а каждый компонент получает доступ к состоянию приложения, наряду с возможностью вызывать действия для изменения состояния, независимо от расположения этого компонента в дереве.
+
+Кроме того, чётко определяя и разделяя концепции, возникающиее при управлении состоянием, и требуя соблюдения некоторых правил, мы улучшаем структурированность и поддерживаемость нашего кода.
```
Comment (gbezyuk) [inline docs/ru/intro.md]: fixed

**Comment by gbezyuk** [comment]
@yyx990803, @chrisvfritz — your turn, guys! =)

**Comment by gbezyuk** [comment]
proof-read by @kojoru and ready for merge

File: docs/ru/forms.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+Если вы используете Vuex в strict mode, привязать `v-model` к состоянию, хранимому во Vue, может быть немного непросто:
+
+``` html
+<input v-model="obj.message">
+```
+
+Предположим, что `obj` — это вычисляемое свойство, возвращающее ссылку на объект из хранилища. В таком случае, `v-model` будет пытаться напрямую изменить значение `obj.message` в ответ на действия пользователя. В strict mode это спровоцирует ошибку, поскольку это изменение происходит вне обработчика мутации Vuex.
+
+Чтобы подружить Vuex с такой ситуацией, следует однонаправленно связать аттрибут `value` элемента `<input>` с полем объекта, а для учёта изменений использовать событие `change`:
```
Comment (ipelekhan) [inline docs/ru/forms.md]: аттрибут => атрибут

File: docs/ru/plugins.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+    // сравнение prevState и nextState...
+
+    // сохранение состояния для следующей мутации
+    prevState = nextState
+  })
+}
+```
+
+**Плагины, снимающие слепки, должны использоваться только на этапе разработки.** При использовании Webpack или Browserify, мы может отдать этот момент на их откуп:
```
Comment (ipelekhan) [inline docs/ru/plugins.md]: > мы может отдать

**Comment by chrisvfritz** [comment]
Awesome! I can confirm it does look like Russian. 😅 @simplesmiler?

File: docs/ru/forms.md (hunk 2 above)
Comment (gbezyuk) [inline docs/ru/forms.md]: fixed

**Comment by simplesmiler** [comment]
Looks good to me! :shipit:

**Pull request merged**